test/unit/generator/content/schedule/part3/index.test.js

```diff
@@ -0,0 +1,76 @@
+jest.mock('../../../../../../app/generator/content/schedule/part3/get-table')
+const { getTable } = require('../../../../../../app/generator/content/schedule/part3/get-table')
+jest.mock('../../../../../../app/generator/content/schedule/part3/get-reduction-summary')
+const getReductionSummary = require('../../../../../../app/generator/content/schedule/part3/get-reduction-summary')
+jest.mock('../../../../../../app/generator/content/schedule/part3/get-top-up-summary')
+const getTopUpSummary = require('../../../../../../app/generator/content/schedule/part3/get-top-up-summary')
+
+const part3 = require('../../../../../../app/generator/content/schedule/part3')
+
+const { topUpSchedule, reductionSchedule } = require('../../../../../mocks/mock-schedule')
+
+describe('schedule part 3', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  test('Ensure top-up summary header style is set to header3', () => {
+    const result = part3(topUpSchedule)
+    expect(result.stack[0].style).toBe('header3')
+  })
+
+  test('Ensure top-up summary title text is "Payment schedule"', () => {
+    const result = part3(topUpSchedule)
+    expect(result.stack[0].text).toBe('Payment schedule')
+  })
+
+  test('Ensure reduction summary header style is set to header3', () => {
+    const result = part3(reductionSchedule)
+    expect(result.stack[0].style).toBe('header3')
+  })
+
+  test('Ensure reduction summary title text is "Payment schedule"', () => {
+    const result = part3(reductionSchedule)
+    expect(result.stack[0].text).toBe('Payment schedule')
+  })
+
+  test('Ensure top-up-schedule returns top-up summary', () => {
```

Comment:
Test names below this don't match what the test is asserting.

Can we change these to "Ensure top-up-schedule is called".

Further, can we add tests that:
1. it is called once
2. it is called with expected values